### PR TITLE
Fix #56: Notebook 5.0 compat

### LINKED ISSUE #56: CLI quick-setup incompatibility with notebook 5.0
The BaseNBExtensionApp class is gone so the check if we're in >=4.2 fails here: https://github.com/jupyter/dashboards/blob/master/jupyter_dashboards/extensionapp.py#L12. Probably need to use notebook.extensions.BaseExtensionApp instead. That or remove the quick-setup convenience step and have people write out the install/enable/disable/uninstall manually.

https://github.com/jupyter/notebook/blob/master/notebook/extensions.py#L52

Ref: https://github.com/jupyter/notebook/issues/2150

### PR DESCRIPTION
**Pull request by parente**
Fixes the install process under notebook 5.0. 

Removes the bundler ability completely in favor of the implementation in notebook 5.0. Doing this forces jupyter/dashboards_bundlers (and other clients of the bundler API) to require notebook>=5.0 or to rely on jupyter_cms<0.7 once this gets released.

### CODE REVIEW

**Comment by parente** [comment]
@jtyberg would you mind taking a quick look see at the final result here to make sure it's not totally off in the weeds?

File: README.md [hunk 1]
```diff
@@ -12,180 +12,94 @@ The content management extensions provide the following:
...
+jupyter nbextension install --py jupyter_cms --sys-prefix
+jupyter nbextension enable --py jupyter_cms --sys-prefix
 ```
 
-In either case, you will need to restart your notebook server if it was running during the enable/activate step. Also, note that you can run `jupyter --paths` to get a sense of where the extension files will be installed.
-
-## Uninstall It
-
-In Jupyter Notebook 4.2:
+Altenratively, the followign command both installs and enables the package
```
Comment (jtyberg) [inline README.md]: Alternatively...following...

File: jupyter_cms/extensionapp.py [hunk 2]
```diff
@@ -186,19 +171,21 @@ def start(self):
...
     """Installs and enables all parts of this extension"""
     name = "jupyter cms quick-setup"
     version = __version__
     description = "Installs and enables all features of the jupyter_cms extension"
 
     def start(self):
         self.argv.extend(['--py', 'jupyter_cms'])
-        
+
+        print('install server')
```
Comment (jtyberg) [inline jupyter_cms/extensionapp.py]: just checking, but keep or remove `print` statement?

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+ipython>=4.1.0
+nbconvert>=5.0.0
+notebook>=4.0.0
```
Comment (jtyberg) [inline requirements.txt]: notebook>=4.2.0?

File: setup.py [hunk 4]
```diff
@@ -14,8 +14,9 @@
     exec(f.read(), {}, VERSION_NS)
 
 install_requires=[
-    'notebook>=4.0.0,<5.0',
     'ipython>=4.1.0',
+    'nbconvert>=5.0.0',
+    'notebook>=4.0.0',
```
Comment (jtyberg) [inline setup.py]: notebook>=4.2.0?

**Comment by jtyberg** [comment]
I tested on notebook 4.4.1 without issue.

Bye-bye bundlers.

File: README.md (hunk 1 above)
Comment (parente) [reply README.md]: Must have been a rough night.

File: jupyter_cms/extensionapp.py (hunk 2 above)
Comment (parente) [reply jupyter_cms/extensionapp.py]: 🔫

**Comment by parente** [comment]
Thanks for the review, @jtyberg

**Pull request merged**